config/local_env.yml

```diff
@@ -1,3 +1,5 @@
 GITHUB_TEMPLATE_REPO_URL: 'philipritchey/autograded-assignment-template'
 GITHUB_COURSE_ORGANIZATION: 'AutograderFrontend'
 ASSIGNMENTS_BASE_PATH: 'assignment-repos/'
+GITHUB_CLIENT_ID: "Ov23liMDAgFqwvCIF4kD"
```

Comment:
We should not be pushing our client secret.

Reply:
Right, fixed that.